sky/packages/sky/example/game/main.dart

```diff
@@ -93,27 +94,37 @@ class GameDemoApp extends App {
     return new Stack([
       new SpriteWidget(new MainScreenBackground()),
       new Center(
```

Comment:
There's no need for the Center widget.  It's redundant with the Flex you've got inside it.